play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -73,6 +73,7 @@ public EbeanConfig parse() {
                 ServerConfig serverConfig = new ServerConfig();
                 serverConfig.setName(key);
                 serverConfig.loadFromProperties();
+                serverConfig.setH2ProductionMode(true);  // Since Ebean 9.1.1: Don't override Evolution
```

Comment:
Do we really need this to be explicitly set here?

I think this should be configurable. Of course, we can set the default to true via [reference.conf](https://github.com/playframework/play-ebean/blob/master/play-ebean/src/main/resources/reference.conf).